### DISCUSSION TITLE
docs/ninjutsu/medium-ranking-ninja/iam-hamsters

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#27] [General]
# docs/ninjutsu/medium-ranking-ninja/iam-hamsters

就像巨龙守护着财宝，如同仓鼠收集松子，恰如河狸收集树木的嫩枝条。

https://996.ninja/docs/ninjutsu/medium-ranking-ninja/iam-hamsters

**Comment by pocket20250310** [comment]
行业报表点不进去 这个网站注册是要邀请码吗？我有github账号 但是怎么试都不行 一直提示密码错误

**Reply by MingGH** [reply]
是的，linux.do 那个网站需要邀请码
